Library/Formula/gcc.rb

```diff
@@ -32,6 +32,15 @@ def osmajor
     sha256 "77bce635f78bc26bd01010b5ece480251af223bf2dba6d48c29af6b29b441296" => :mountain_lion
   end
 
+  if MacOS::Xcode.version > "7.0" || MacOS::CLT.version > "7.0"
```

Comment:
Seems to be Xcode version specific; people running Xcode 6 on El Capitan have reported it works (although everything else is broken). I strongly suspect people running Xcode 7 beta on Yosemite will have the same issues.